kong/dao/migrations/cassandra.lua

```diff
@@ -423,4 +423,27 @@ return {
     end,
     down = function(_, _, dao) end
   },
+  {
+    name = "2017-07-31-113200_jwt_preflight_default",
+    up = function(_, _, dao)
+      local rows, err = dao.db:query([[
+        SELECT * FROM plugins WHERE name = 'jwt';
```

Comment:
This is plugin configuration, which is in the core. The plugins own `daos.lua` file is what the plugin specific migrations target.

Do you want plugin specific migrations to start updating core entities?